NPE when running -c with -s args

```
~/workspace/deleteme ❯ cat hello.jsh
println("hello")%
~/workspace/deleteme ❯ jbang -c -s=hello.jsh --verbose
[jbang] jbang version 0.90.0
[jbang] [ERROR] public class java.lang.NullPointerException
java.lang.NullPointerException
	at dev.jbang.util.Util.getStableID(Util.java:916)
	at dev.jbang.source.resolvers.LiteralScriptResourceResolver.stringToResourceRef(LiteralScriptResourceResolver.java:51)
	at dev.jbang.cli.Run.doCall(Run.java:97)
	at dev.jbang.cli.BaseCommand.call(BaseCommand.java:93)
	at dev.jbang.cli.BaseCommand.call(BaseCommand.java:15)
	at picocli.CommandLine.executeUserObject(CommandLine.java:1953)
	at picocli.CommandLine.access$1300(CommandLine.java:145)
	at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2352)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2346)
	at dev.jbang.cli.JBang$3.handle(JBang.java:145)
	at dev.jbang.cli.JBang$3.handle(JBang.java:140)
	at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2179)
	at picocli.CommandLine.execute(CommandLine.java:2078)
	at dev.jbang.Main.main(Main.java:14)
[jbang] If you believe this a bug in jbang, open an issue at https://github.com/jbangdev/jbang/issues
~/workspace/deleteme ❯ jbang -c 'println("hello")'
hello

~/workspace/deleteme ❯ jbang hello.jsh
hello
```

--- a/src/main/java/dev/jbang/cli/Run.java
+++ b/src/main/java/dev/jbang/cli/Run.java
@@ -73,7 +73,8 @@ public class Run extends BaseBuildCommand {
 
 	@Override
 	public Integer doCall() throws IOException {
-		if (scriptOrFile == null && !interactive && !literalScript.isPresent()) {
+		if (scriptOrFile == null && ((!interactive && !literalScript.isPresent())
+				|| (literalScript.isPresent() && literalScript.get().isEmpty()))) {
 			throw new IllegalArgumentException("Missing required parameter: '<scriptOrFile>'");
 		}
 

